test/functional/sqlsrv/sqlsrv_ae_output_param_sqltype_numeric.phpt

```diff
@@ -101,24 +101,26 @@ foreach ($dataTypes as $dataType) {
                         if (stripos("SQLSRV_SQLTYPE_" . $dataType, $sqlType) !== false) {
                             var_dump(sqlsrv_errors());
                             $success = false;               
-                        }	
+                        }
                     }
                     else {
-                        if ($dataType == "float" || $dataType == "real") {
-                            if (abs($c_detOut - $inputValues[0]) > $epsilon || abs($c_randOut - $inputValues[1]) > $epsilon) {
-                                echo "Incorrect output retrieved for datatype $dataType and sqlType $sqlType:\n";
-                                print("    c_det: " . $c_detOut . "\n");
-                                print("    c_rand: " . $c_randOut . "\n");
-                                $success = false;
-                            }
-                        } else {
-                            if ($c_detOut != $inputValues[0] || $c_randOut != $inputValues[1]) {
-                                echo "Incorrect output retrieved for datatype $dataType and sqlType $sqlType:\n";
-                                print("    c_det: " . $c_detOut . "\n");
-                                print("    c_rand: " . $c_randOut . "\n");                                
-                                $success = false;
-                            }
-                        }			
+                        if (AE\IsDataEncrypted() || stripos("SQLSRV_SQLTYPE_" . $dataType, $sqlType) !== false) {

```

Comment:
can you put a comment here saying this condition is needed to compare input and output values when data encrypted is false